### PR TITLE
chore: add zombienet config files for quick spin-ups

### PR DESCRIPTION
**Pull request by ntn-x2**
Quick way of spinning up a parachain instance using actual node software and consensus rather than mocking stuff with Chopsticks.

### CODE REVIEW

**Comment by ntn-x2** [comment]
@Ad96el I am merging this and the Chopsticks PRs, otherwise testing any new features is extremely complicated and requires constant branch switching. Please still provide a review when you have time, and any comments will be addressed in a follow-up PR, thank you!

File: zombienet/runtimes/spiritnet/polkadot-local-fast-v1.3.2-e220854.json
Comment (Ad96el) [inline zombienet/runtimes/spiritnet/polkadot-local-fast-v1.3.2-e220854.json:1]: How do we update those chain specs? For the release, we are normally trying to be as close as possible to the current running polkadot version.  Could you maybe document that in the readme?

**Review by Ad96el — COMMENTED** [review]
LOVE IT :heart_eyes:

File: zombienet/runtimes/spiritnet/polkadot-local-fast-v1.3.2-e220854.json
Comment (ntn-x2) [reply zombienet/runtimes/spiritnet/polkadot-local-fast-v1.3.2-e220854.json:1]: Good point. I made a ticket for it: https://github.com/KILTprotocol/ticket/issues/3633

**Pull request merged**